sdk/azidentity/live_test.go

```diff
@@ -111,6 +141,8 @@ func TestMain(m *testing.M) {
 			liveSP.tenantID:                                 fakeTenantID,
 			liveUser.tenantID:                               fakeTenantID,
 			liveUser.username:                               fakeUsername,
+			adfsLiveSP.tenantID:                             fakeTenantID,
+			adfsLiveUser.tenantID:                           fakeTenantID,
```

Comment:
We don't want to replace "adfs" as the tenant in recordings because we want to test any magic around this value, and we don't need to replace it to make playback work because it's a static string.